scripts/hi_res_model_initialize.py

```diff
@@ -0,0 +1,16 @@
+import os
+
+from unstructured_inference.models.base import get_model
+
+
+def initialize():
+    """Download default model or model specified by UNSTRUCTURED_HI_RES_MODEL_NAME environment
+    variable (avoids subprocesses all doing the same)"""
+
+    # If more than one model will be supported and left up to user selection
+    supported_model = os.environ.get("UNSTRUCTURED_HI_RES_SUPPORTED_MODEL", "")
+    if supported_model:
+        for model_name in supported_model.split(","):
+            get_model(model_name=model_name)
+
+    get_model(os.environ.get("UNSTRUCTURED_HI_RES_MODEL_NAME"))
```

Comment:
This is fine for the quick workaround, but it seems like this initialization code should be moved within unstructured itself, out of the ingest module, maybe into its own top-level module.

Reply:
yeah i was thinking maybe under `partition` dir?

Reply:
that works